frontend/src/app/projects/models/diagrams/model-diagram-preview-dialog/model-diagram-preview-dialog.component.html

```diff
@@ -14,6 +14,6 @@ <h2>Diagram preview of '{{ data.diagram.name }}'</h2>
   </div>
   <div class="width">
     <mat-divider id="bottom-divider"></mat-divider>
-    <button mat-button mat-dialog-close>Close</button>
+    <button class="width" mat-button mat-dialog-close>Close</button>
```

Comment:
It could use a more descriptive class name, e.g. `wide`.

Reply:
Will rename it.

Reply:
Is `w-full` (tailwind) now.